src/input/mod.rs

```diff
@@ -2344,17 +2380,29 @@ fn should_intercept_key(
                     repeat: true,
                     cooldown: None,
                     allow_when_locked: false,
+                    // The screenshot UI owns the focus anyays, so this doesn't really matter?
+                    // But logically, nothing can inhibit its actions. Only opening it can be
+                    // inhibited.
```

Comment:
```suggestion
                    // The screenshot UI owns the focus anyway, so this doesn't really matter.
                    // But logically, nothing can inhibit its actions. Only opening it can be
                    // inhibited.
```
Also yeah, either way it's better to safeguard against errors by passing false